api-contracts/openapi/components/schemas/monitoring.yaml

```diff
@@ -0,0 +1,4 @@
+RunProbe:
```

Comment:
It looks like this object is just returning the `tenantId` which is getting passed in anyway, is there a reason we're returning this object?

Reply:
nope was just getting errors if I removed it, didn't realize I could just not return an object

Reply:
Still looks like this is here, we should be good to delete this file along with `MonitoringRunProbe` defined in `_index.yaml`